Practice2/src/partials/top-section.html

```diff
@@ -5,39 +5,39 @@
     <source media="(min-width: 320px)" srcset="./assets/images/top-background-mobile.png">
     <img src="./assets/images/top-background-mobile.png" alt="background" class="top-img">
   </picture>
-    <div class="nav-top">
-      <p class="top-text">h.</p>
-      <ul class="nav">
-        <li class="nav-item">
-          <a class="nav-link nav-link-success" href="javascript:void(0)">Hotels</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link nav-link-success" href="javascript:void(0)">Airlines</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link nav-link-success" href="javascript:void(0)">Vacations</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="javascript:void(0)">Find more</a>
-        </li>
-      </ul>
-      <div class="nav-icon">
-        <img src="./assets/images/twitter-icon.png" alt="twitter-icon" class="nav-icon-success">
-        <img src="./assets/images/facebook-icon.png" alt="facebook-icon" class="nav-icon-success">
-        <img src="./assets/images/instagram-icon.png" alt="instagram-icon">
-      </div>
-      <button class="nav-menu"><img src="./assets/images/menu-icon.png" alt="menu-icon"></button>
+  <div class="nav-top">
```

Comment:
If this block is a navigation, why don't we use `nav` instead?

Reply:
I still don't understand what you mean

Reply:
I still don't understand what you mean

Reply:
@duylong13 Pls try to search before making the statement "I don't understand". HTML `nav` tag for a navigation block is the basic knowledge that you already learned.

Reply:
Ahhh,  I understand. The nav tag I originally gave the common-component was based on the name of the document Vuong gave me. So I took that part and put it in my homework